sgrep/bin/main_sgrep.ml

```diff
@@ -423,6 +427,7 @@ let options () =
     "-rules_file", Arg.Set_string rules_file,
     " <file> obtain list of patterns from YAML file";
 
+    "-json", Arg.Set output_format_json, "output to JSON";
```

Comment:
@aryx I couldn't find an easy way to make the one_pattern also output json. Seems some refactoring of the output logic would be useful to make this work. Difficult because of the stream vs collecting output model between the lint vs not lint approach right now.